pkg/build/admission/defaults/api/types.go

```diff
@@ -25,6 +25,9 @@ type BuildDefaultsConfig struct {
 	// SourceStrategyDefaults are default values that apply to builds using the
 	// source strategy.
 	SourceStrategyDefaults *SourceStrategyDefaultsConfig
+
+	// Resources computes resource requirements to execute the build.
+	Resources kapi.ResourceRequirements
```

Comment:
and then you will need to add a test where Resources is nil (in addition to the existing test where Resources is non-nil but empty)